impl/src/main/java/org/ehcache/impl/internal/store/offheap/OffHeapMap.java

```diff
@@ -0,0 +1,23 @@
+package org.ehcache.impl.internal.store.offheap;
+
+/**
+ * Off-heap-backed maps counters.
+ *
+ * @author Ludovic Orban
+ */
+public interface OffHeapMap {
```

Comment:
Just to avoid polluting the `EhcacheOffHeapBackingMap` interface with stats concerns. But I agree, at the very least this new interface is poorly named.